src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars--star"></div>
+      <div class="stars--star"></div>
+      <div class="stars--star"></div>
+      <div class="stars--star"></div>
+      <div class="stars--star"></div>
+    </div>
+
+    <div class="stars stars--1">
+      <div class="stars--star stars--N"></div>
+      <div class="stars--star"></div>
+      <div class="stars--star"></div>
+      <div class="stars--star"></div>
+      <div class="stars--star"></div>
+    </div>
+
+    <div class="stars stars--2">
+      <div class="stars--star stars--N"></div>
```

Comment:
```suggestion
      <div class="stars--star"></div>
```
rewrite your code without using the additional classes

Reply:
It's just written in the task to use stars--N

Reply:
Did I understand correctly, should I write everything without class stars--N or just this part, line 33?